mongo/integration/initial_dns_seedlist_discovery_test.go

```diff
@@ -199,3 +221,14 @@ func getServerByAddress(address string, topo *topology.Topology) (description.Se
 	defer selectedServerConnection.Close()
 	return selectedServerConnection.Description(), nil
 }
+
+func countAllServers(topo *topology.Topology) int {
```

Comment:
I wrote this function to help assert [`numHosts`](https://github.com/mongodb/specifications/pull/1069/files#diff-ff35a400471d3661bc790af0f67d3c963e6b269dad31aa6c8cfe96af9c970fbdR96-R98).

> [`numHosts` is] the expected number of hosts discovered once SDAM completes a
  scan.

I took this to mean the number of candidate servers returned in a simple server selection, and I wrote this strange server selector below to save `len(servers)`. Let me know if this doesn't make sense.

Reply:
Can we not use `Topology#Description` here and count the hosts from that?